### PR TITLE
Fixes #510: Improve infinite pagination in media wall

### LINKED ISSUE #510: Improve infinite pagination in media wall
Pagination in media wall can be implemented using `since` and `until` field of loklak search.json API.

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
- Improve pagination using `since` and `until` field of loklak server

**TODO**
- Removing some duplicate feeds 
- Implement it using the timestamp of the last feed in the next PR. 
- Also, I need the suggestion that as the number of feeds increase, speed slows down. How to improve it.

This is beneficial for loklak media wall because the number of feeds on pagination can be undetermined.

**Closes #510 **

### CODE REVIEW

**Comment by Achint08** [comment]
Please review @hemantjadon @djmgit @singhpratyush @vibhcool @kavithaenair @SKrPl.

**Comment by singhpratyush** [comment]
And all the elements are refreshed (fade out then fade in) when new Tweets come up. This is distracting for someone who reads the Tweets.

**Comment by vibhcool** [comment]
@singhpratyush, I noticed that the date formats you mentioned are working on Twitter but not on search endpoint. Could you confirm this?

@Achint08 , to maintain uniformity, we shall date format `2017-08-02T21:54:42.000Z` everywhere.

**Comment by hemantjadon** [comment]
@singhpratyush @vibhcool The time formats mentioned are there in the documentation. Do docs need a review?

File: src/app/reducers/api-media-wall-response.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export function reducer(state: State = initialState, action: apiAction.Actions |
 
 		case wallPaginationAction.ActionTypes.WALL_PAGINATION_COMPLETE_SUCCESS: {
 			const apiResponse = action.payload;
+			console.log(apiResponse.statuses.length);
```
Comment (hemantjadon) [inline src/app/reducers/api-media-wall-response.ts]: Remove this console

**Comment by Achint08** [comment]
@hemantjadon @vibhcool @singhpratyush check out now. :)
https://achint08.github.io/loklak_search/wall?query=fossasia

**Pull request closed without merging**